FabricTestExample/src/Test654.js

```diff
@@ -5,9 +5,15 @@ import { createNativeStackNavigator } from '@react-navigation/native-stack';
 
 const Stack = createNativeStackNavigator();
 
-I18nManager.forceRTL(true);
-
 export default function App() {
+  I18nManager.forceRTL(true);
+
+  React.useEffect(() => {
+    return () => {
+      I18nManager.forceRTL(false);
+    };
+  });
```

Comment:
Shouldn't it be put inside useEffect?

Reply:
Moved in https://github.com/software-mansion/react-native-screens/pull/2130/commits/0d3f60080b1e931ccc0d417442ea1f833d3e039e.

Reply:
Now you could convert it into a hook to simplify the code :D

Reply:
Makes no sense just for two tests to be honest, but let's remember about it when we'll add more tests 👍